docs_src/microservices/general/messagebus.md

```diff
@@ -249,7 +249,7 @@ All EdgeX services are capable of using MQTT 3.1 by simply making changes to eac
 
 #### Docker
 
-The EdgeX Compose Builder utility provides an option to easily generate a compose file with all the selected services re-configured for MQTT 3.1 using environment overrides. This is accomplished by using the `mqtt-bus` option. See [Compose Builder README](https://github.com/edgexfoundry/edgex-compose/tree/{{latest_release_name}}/compose-builder) for details on all available options.
+The EdgeX Compose Builder utility provides an option to easily generate a compose file with all the selected services re-configured for MQTT 3.1 using environment overrides. This is accomplished by using the `mqtt-bus` option. See [Compose Builder README](https://github.com/edgexfoundry/edgex-compose/tree/{{version}}/compose-builder) for details on all available options.

```

Comment:
I've used version instead of release name because that would keep the link target code consistent with the docs. Untagged dev changes to a branch will not become reachable from the published docs.

Reply:
This was done this way because currently the release branch (levski) has the latest changes for patched device and app services. These are not available via the tag (2.3.0). Also we have not created patch versions of the documentation when we create a full EdgeX patch release, which would result in a patch tag of edgex-compose.

I have asked James to add this to this week's TSC agenda.

Reply:
The documentation is versioned on major.minor releases but the changes in question are for a patch. That's not an issue. To address it, the `version` variable in the template will need to be bumped to 2.3.1 patch upon release without the need to tag the documentation itself.

If you insist on having release names (which I think are problematic as explained in the last comment): this is the main documentation branch and its `release` variable should be set to `main` now and `minnesota` upon release. In this case, the release name of `levski` branch would be `levski`.

Reply:
> if you insist on having release names (which I think are problematic as explained in the last comment): this is the main documentation branch and its `release` variable should be set to `main` now and `minnesota` upon release. In this case, the release name of `levski` branch would be `levski`.

Agreed

Reply:
Not sure which part you agreed on. I still do think having a release name variable is neither appropriate nor necessary. Maybe I'm wrong.

I added some comments to the macros file to clarify my points.

Reply:
Need to also keep in mind that the version is only useful if all repos are tagged consistently. I just noticed v2.3.1 tags on some repos but not the others.

Reply:
That is because App and Device service can have patch releases w/o triggering a patch release for all services/repos. I.e. device SDK and camera service just release 2.3.1 patch.